samples/snippets/src/main/java/com/example/spanner/HubbleTransactionsCodeLab.java

```diff
@@ -14,18 +14,21 @@
 
+  private static final String SUBJECT = "Random subject for the email for testing purpose %s";
+
   public void createMessages(DatabaseAdminClient dbAdminClient, DatabaseId id) {
     createDatabase(
         dbAdminClient,
         id,
-        Arrays.asList(
+        Collections.singletonList(
```

Comment:
Our external documentation uses Arrays.asList. What is the benefit of singletonList?

Reply:
Basically Collections.singletonList is immutable with  single element vs Arrays.asList is mutable expecting other elements to be added in the list. Here we know we are not going to modify the list that is why use it. Else both works fine.

Reply:
Let's stick to Arrays.asList. I agree that Collections.singletonList makes sense, but I'd prefer to be consistent with our external samples.